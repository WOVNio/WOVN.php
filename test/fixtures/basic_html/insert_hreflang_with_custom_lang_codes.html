<!DOCTYPE html>
<html>
<body style="container">
<p>Fake body</p>
</body>
</html>
