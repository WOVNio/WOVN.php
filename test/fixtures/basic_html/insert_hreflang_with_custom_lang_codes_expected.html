<!DOCTYPE html>
<html>
<body style="container"><link rel="alternate" hreflang="en" href="http://my-site.com/"><link rel="alternate" hreflang="vi" href="http://my-site.com/vi/"><link rel="alternate" hreflang="zh-Hans" href="http://my-site.com/custom_simple/"><script src="//j.wovn.io/1" data-wovnio="key=123456&amp;backend=true&amp;currentLang=en&amp;defaultLang=en&amp;urlPattern=path&amp;langCodeAliases={&quot;zh-CHS&quot;:&quot;custom_simple&quot;}&amp;langParamName=wovn" data-wovnio-info="version=WOVN.php_VERSION" async></script>
<p>Fake body</p>
</body>
</html>
